app/templates/components/book-cover.hbs

```diff
@@ -1,7 +1,11 @@
-<li {{action "open"}}>
-  <strong>{{book.title}}</strong><br><br>
+<li >
+  <strong{{action "open"}}>{{book.title}}</strong><br><br>
```

Comment:
Nice use of actions here!

We could improve accessibility here by adding a `role="button"` to this element.

However, I would personally remove these `br`'s, change from `strong` -> `div` and  use a class and some CSS to accomplish same thing